README.md

```diff
@@ -16,6 +16,7 @@ Contributions to this repo are expected to adhere to the [Biconomy Solidity Styl
 ### Build
 
 ```shell
+$ git clone git@github.com:BreadchainCoop/breadchain.git --recursive 
```

Comment:
Probably an unnecessary line because we can assume a user knows how to use git. Otherwise, I'd move this to a separate section called "Clone".